README.md

```diff
@@ -1,25 +1,38 @@
+[![SublimeHQ Discord](https://img.shields.io/discord/280102180189634562?label=SublimeHQ%20Discord&logo=discord&style=flat-square)](https://discord.gg/D43Pecu)
+
 # Sublime Packages
 
-> **These packages are developed against the latest [Sublime Text 3 Dev Build](http://sublimetext.com/3dev). Bugs may exist on older builds, and the format used is not compatible with builds older than 3092.**
+> **These packages are developed against the latest [Sublime Text Dev Build](http://sublimetext.com/).**
+> **Bugs may exist on older builds or even cause crashes, and the format used is not compatible with builds older than 4094.**
 
 ## Installation
 
-If you want to make changes to these packages and test them locally, fork this repository and then symlink the changed packages into your *Packages* folder.
+If you want to make changes to these packages and test them locally, fork this repository and then symlink the changed packages into your *`Packages`* folder.
 
 *Replace `Python` in the following commands with the name of the syntax to install.*
 
+### Sublime Text 3 Users
+
+Please use the `st3` branch:
+
+<https://github.com/sublimehq/Packages/tree/st3>
+
+and change the paths below accordingly as described in the README there:
+
+<https://github.com/sublimehq/Packages/tree/st3#readme>
+
 ### OS X
 
 ```bash
 $ git clone https://github.com/sublimehq/Packages.git
-$ ln -s `pwd`/Packages/Python ~/Library/Application\ Support/Sublime\ Text\ 3/Packages/
+$ ln -s `pwd`/Packages/Python ~/Library/Application\ Support/Sublime\ Text/Packages/
```

Comment:
The directions for all three platforms here will be wrong if they upgraded from an ST3 install, since it will keep using their ST3 data dir. It really will need them to look for their data dir. Probably the easiest way is to have them use *Preferences > Browse Packages…*.